src/compiler/crystal/codegen/primitives.cr

```diff
@@ -18,6 +18,9 @@ class Crystal::CodeGenVisitor
             when "binary"
               codegen_primitive_binary node, target_def, call_args
             when "cast"
+              # TODO 0.28.0 delete :cast
+              codegen_primitive_cast node, target_def, call_args
+            when "convert"
```

Comment:
Just use `when "cast", "convert"`?

Reply:
The convert case will have some changes in the overflow branch and is to flag that the cast should be removed